recipes-halif/devicesettings/devicesettings-hal-raspberrypi4.bb

```diff
@@ -6,7 +6,7 @@ SECTION = "console/utils"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
 
-PROVIDES = "virtual/devicesettings-hal"
+PROVIDES = "virtual/devicesettings-hal virtual/vendor-devicesettings-hal"
 RPROVIDES_${PN} = "virtual/devicesettings-hal"
```

Comment:
Line 10 : Here also don't we need virtual/vendor-devicesettings-hal added ?

Reply:
https://github.com/rdkcentral/meta-rdk-raspberrypi/pull/64